src/olympia/reviewers/templates/reviewers/review.html

```diff
@@ -300,7 +300,14 @@ <h3 id="history">
               <button data-api-url="{{ drf_url('reviewers-addon-flags', addon.pk) }}"
                       data-api-method="patch"
                       data-api-data="{&quot;pending_info_request&quot;: null}"
-                      id="clear_pending_info_request" type="button">{{ _('Clear information request') }}</button>
+                      id="clear_pending_info_request" class="oneoff" type="button">{{ _('Clear information request') }}</button>
+          </li>
+        {% endif %}
+
+        {% if addon.versions.filter(reviewerflags__pending_rejection__isnull=False) %}
```

Comment:
can we move this query to a template variable.  I know we _can_ write almost anything in jinja templates, just not sure we should.

Reply:
https://github.com/mozilla/addons-server/pull/14837/commits/ef139efa0469eee776009558c46958d6a6015c2b